index.js

```diff
@@ -3,81 +3,81 @@ const addButton = document.querySelector('#add');
 const titleInput = document.querySelector('#title');
 const autherInput = document.querySelector('#author');
 
-// Create a collection that keeps a list of books
-const listOfBooks = [
-  {
-    title: 'atomic habit',
-    author: 'James Clear',
-  },
-  {
-    title: 'Rich dad poor dad',
-    author: 'Robert kiosaki',
-  },
-  {
-    title: 'Think and grow rich',
-    author: 'napleon hill',
-  },
-];
+// define class with object constructor
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
 
-// Create a function to add a new book to the collection, with title and author
-function addBook(title, author) {
-  const tempBook = {
-    title,
-    author,
-  };
-  // save the new added book into the collection
-  listOfBooks.push(tempBook);
-  // desplay the new added book into the page
-  const addedBooks = document.querySelector('#add_books');
-  const book = document.createElement('article');
-  const titl = document.createElement('p');
-  titl.innerText = tempBook.title;
-  const authr = document.createElement('p');
-  authr.innerText = tempBook.author;
-  book.append(titl);
-  book.append(authr);
-  const button = document.createElement('button');
-  button.innerText = 'Remove';
-  button.addEventListener('click', () => {
-    book.style.display = 'none';
-  });
-  book.append(button);
-  const line = document.createElement('hr');
-  book.append(line);
-  addedBooks.append(book);
-}
+  static count = 0;
+
+  static listOfBooks = [];
 
-// Create a function to remove a book from the collection
-function removeBook(index) {
-  listOfBooks.splice(listOfBooks.findIndex((e) => e.title === listOfBooks[index].title
-  && e.author === listOfBooks[index].author), 1);
+  static addBook(title, author) {
```

Comment:
- [ ] The functionality of adding a book to local storage isn't working properly. When a new book is added, it replaces the previously stored book in local storage instead of appending the new book to local storage. Kindly see my video below for more explanation:

https://www.loom.com/share/a3293800328e4c40a430302d931b37c8